app/concepts/todo/operation.rb

```diff
@@ -0,0 +1,50 @@
+class Todo::Create < Trailblazer::Operation
+  include Model
+  model Todo, :create
+
+  contract do
+    property :title, validates: { presence: true }
+    property :description
+    property :todo_list_id
+    property :completed
+  end
+
+  def process(params)
+    # validate and save
+    validate(params[:todo]) do
+      contract.save
+    end
+  end
+
+end
+
+class Todo::Update < Trailblazer::Operation
+  include Model
+  model Todo, :update
+
+  contract do
+    property :title, validates: { presence: true }
+    property :description
+    property :completed
+  end
+
+  def process(params)
+    # validate and save
+    validate(params[:todo]) do
+      contract.save
+    end
+  end
+
+end
+
+
+class Todo::Delete < Trailblazer::Operation
+  include Model
+  model Todo, :find
```

Comment:
`:find` will execute Model.find(param[:id]), http://trailblazer.to/gems/operation/model.html